classes/mender-image-buildinfo.bbclass

```diff
@@ -1,15 +1,13 @@
-inherit image-buildinfo
-
 DEVICE_TYPE ?= "${MACHINE}"
 IMAGE_ID = "${PN}-${DATETIME}"
-IMAGE_BUILDINFO_VARS = "DISTRO DATETIME PN IMAGE_ID DEVICE_TYPE"
 
 buildinfo_mender () {
 cat > ${IMAGE_ROOTFS}${sysconfdir}/build_mender << END
```

Comment:
Yes, that's preexisting, I agree it has a horrible name, but I won't handle that in the PR.